inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
 
+    def _is_reboot_device(self) -> bool:
+        return self._reboot_device not in ["No", "N", "n", "no", "NO"]
+
     def check(self) -> None:
         """Perform manifest checking before SOTA"""
         logger.debug("")
```

Comment:
**<h2>$\color{#15F4EE}Miscellaneous$</h2>**
**Suggestion:** Refactor the check method to ensure that it uses the updated variable names and logic after the refactor.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def check(self) -> None:
        # ... updated method implementation ...
```